### PR TITLE
optimize customer important prometheus queries

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/prometheus/configmap.yaml [hunk 1]
```diff
@@ -53,6 +53,17 @@ data:
       - record: job:skipper_average_latency:sum
         expr: sum(rate(skipper_serve_host_duration_seconds_sum{}[1m])) / sum(rate(skipper_serve_host_duration_seconds_count{}[1m]))
 
+    - name: skipper sum rate serve_host_duration by host,bucket
+      rules:
+      - record: job:skipper_serve_host_duration_host_bucket:sum
+        expr: sum(rate(skipper_serve_host_duration_seconds_bucket{}[1m])) by (le,host)
```
Comment (muaazsaleem) [inline cluster/manifests/prometheus/configmap.yaml:59]: promql question, isn't `sum(rate(skipper_serve_host_duration_seconds_bucket{}[1m])` == `skipper_serve_host_duration_seconds_bucket{}[1m]`

Comment (szuecs) [reply cluster/manifests/prometheus/configmap.yaml:59]: I used the docs to create the query in the past, so I would say no.
`sum(rate(..` is the base of all histogram queries

**Comment by muaazsaleem** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**